config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -232,6 +232,74 @@ periodics:
           cpu: 2
           memory: "2Gi"
 
+- name: ci-kubernetes-kubemark-100-gce-scheduler-highqps
+  tags:
+  - "perfDashPrefix: kubemark-100Nodes-scheduler-highqps"
+  - "perfDashJobType: performance"
+  interval: 24h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-dind-enabled: "true"
+    preset-e2e-kubemark-common: "true"
+    preset-e2e-scalability-periodics: "true"
+    preset-e2e-scalability-periodics-master: "true"
+  annotations:
+    testgrid-dashboards: sig-scalability-scheduler
+    testgrid-tab-name: kubemark-100-highqps
+    testgrid-num-failures-to-alert: '1'
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20201111-a263fd7-master
+      args:
+      - --repo=k8s.io/kubernetes=master
+      - --repo=k8s.io/perf-tests=master
+      - --root=/go/src
+      - --timeout=260
```

Comment:
Seems fairly high - how long it really takes (it's BTW same comment for the job above).

Reply:
the average seems to be around 1 hour (occasionally taking 1-2 hours)

my reasoning behind 260 was: 20 mins to set up the job + 4*60 mins to for each test in the scheduler suite -- none of the tests in the suite should take 60 mins with just 100 nodes, so there was also some breathing space built into the 260

I'm fine with reducing this and the job's timeout to 150 mins

Reply:
Updated both jobs to 150 minutes.